drivers/espi/espi_mchp_xec.c

```diff
@@ -505,6 +506,7 @@ static int espi_xec_send_oob(const struct device *dev,
 	ESPI_OOB_REGS->TX_LEN = pckt->len;
 	ESPI_OOB_REGS->TX_CTRL = MCHP_ESPI_OOB_TX_CTRL_START;
 	LOG_DBG("%s %d", __func__, ESPI_OOB_REGS->TX_LEN);
+	data->oob_sync_pending = true;
```

Comment:
variables using in both ISR and thread(process) context shoudlnt be atomic?

And also it is ideal to update the flag before tx initiation( @ line502) so that variable set event if ISR is too fast.